index.html

```diff
@@ -43,6 +43,105 @@ <h2 class="jobtitle">Software Developer</h2>
         </nav>
 
         </section>
+        <section>
+          <div class="uppergrid">
+            <nav class="work-arrow">
+              <ul class="ul-work">
+                <li><a href="#"><img src="image/arrowdown.png" alt="github"></a></li>
+                <li><a href="#">Work</a></li>
+              </ul>
+            </nav>
+            <h1 class="my-work">MY RECENT WORK</h1>
+            <img src="image/work-divider.png" class="img-workDivider" alt="divider image">
+            </div>
+
+          <div class="workedOn">
+            <div class="project-grid">
+              <div class="grid-item"><img src="image/Img-Placeholder.png" alt="image of project"></div>
+              <div class="grid-item"><h2>Multi-Post Story</h2></div>
+              <div class="grid-item holdtext">
+                A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. 
+                has been the industry's standard dummy text ever since the 1500s, 
+                when an unknown printer took a standard dummy text.
+              </div>
+              <div class="grid-item navlang">
+                <a href="#" class="subgrid-item">CSS</a>
+                <img src="image/css-divider.png" alt="divider image" class="subgrid-item">
+                <a href="#" class="subgrid-item">html</a>
+                <img src="image/css-divider.png" alt="divider image" class="subgrid-item">
+                <a href="#" class="subgrid-item">bootstrap</a>
+                <img src="image/css-divider.png" alt="divider image" class="subgrid-item">
+                <a href="#" class="subgrid-item">Ruby</a>
+              </div>
```

Comment:
- [ ] Kindly use `ul` to list the project technologies (e.g., "JavasScript", "HTML" and "CSS") and each of them a `li` as this is a core project requirement.